core/src/main/java/org/zenoss/zep/index/impl/EventIndexerImpl.java

```diff
@@ -305,12 +310,17 @@ public void handleDeleted(String uuid) throws Exception {
 
             @Override
             public void handleComplete() throws Exception {
+                indexDao.commit();
                 if (calledStartBatch.get()) {
                     for (EventPostIndexPlugin plugin : plugins) {
-                        plugin.endBatch(context);
+                        try {
+                            plugin.endBatch(context);
+                        } catch (Exception e) {
+                            // Post-processing plug-in failures are not fatal errors.
+                            logger.warn("Failed to finish batch for post-processing plug-in.", e);
+                        }
```

Comment:
This is identical to other plugin-calling code, we just weren't handling errors on batch completion before. This probably hasn't actually bit us in the field, but with the Fanout plugin using this method it definitely could.